--- parcellite-0.9.1/data/parcellite.desktop.in.old	2009-12-10 00:16:44.000000000 +0100
+++ parcellite-0.9.1/data/parcellite.desktop.in	2009-12-10 00:24:17.000000000 +0100
@@ -1,9 +1,8 @@
 [Desktop Entry]
-Encoding=UTF-8
-_Name=Parcellite
-_Comment=Clipboard Manager
+Name=Parcellite
+Comment=Clipboard Manager
 Icon=gtk-paste
 Exec=parcellite
 Terminal=false
 Type=Application
-Categories=GTK;GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;
--- parcellite-0.9.1/data/parcellite-startup.desktop.in.old	2009-12-10 00:24:52.000000000 +0100
+++ parcellite-0.9.1/data/parcellite-startup.desktop.in	2009-12-10 00:25:09.000000000 +0100
@@ -1,7 +1,6 @@
 [Desktop Entry]
-Encoding=UTF-8
-_Name=Parcellite
-_Comment=Clipboard Manager
+Name=Parcellite
+Comment=Clipboard Manager
 Icon=gtk-paste
 Exec=parcellite
 Terminal=false
