--- parcellite-0.9.2/data/parcellite.desktop.in.old	2010-01-09 02:37:14.000000000 +0100
+++ parcellite-0.9.2/data/parcellite.desktop.in	2010-01-09 02:37:52.000000000 +0100
@@ -1,8 +1,8 @@
 [Desktop Entry]
-_Name=Parcellite
-_Comment=Clipboard Manager
+Name=Parcellite
+Comment=Clipboard Manager
 Icon=gtk-paste
 Exec=parcellite
 Terminal=false
 Type=Application
-Categories=GTK;GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;
--- parcellite-0.9.2/data/parcellite-startup.desktop.in.old	2010-01-09 02:38:20.000000000 +0100
+++ parcellite-0.9.2/data/parcellite-startup.desktop.in	2010-01-09 02:38:37.000000000 +0100
@@ -1,6 +1,6 @@
 [Desktop Entry]
-_Name=Parcellite
-_Comment=Clipboard Manager
+Name=Parcellite
+Comment=Clipboard Manager
 Icon=gtk-paste
 Exec=parcellite
 Terminal=false
